cppcache/src/ExpiryTaskManager.cpp

```diff
@@ -61,20 +61,6 @@ long ExpiryTaskManager::scheduleExpiryTask(ACE_Event_Handler* handler,
   return m_reactor->schedule_timer(handler, 0, expTimeValue, intervalValue);
 }
 
-long ExpiryTaskManager::scheduleExpiryTask(
```

Comment:
I thought we were trying to get rid of the other scheduleExpiryTask above that took `uint32_t`?

Reply:
Still some internal usage. Focusing on external APIs first.